frontmacs-keys.el

```diff
@@ -87,8 +87,15 @@
 ;; Rename the current buffer and its visiting file if any.
 (global-set-key (kbd "C-c r") 'crux-rename-buffer-and-file)
 
-;; Kill whole line
-(global-set-key (kbd "C-w") 'crux-kill-whole-line)
+;; Kill whole line OR region
+(defun kill-region-or-line ()
+  "Kill the region if active. Else, kill the line."
+  (interactive)
+  (if (region-active-p)
```

Comment:
Maybe use `use-region-p` instead? If the regision is active, but empty, then we probably want to still operate on the line